### PR TITLE
uploading LINUX_MOUNT extension

### PR DESCRIPTION
**Pull request by drapl0n**

### CODE REVIEW

File: payloads/extensions/linux_mount.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# LINUX_MOUNT v1 by @drapl0n
+# Auto mounts BashBunny on GNU/Linux systems.
+# NOTE: Mount path is stored in variable "lmnt".
+# Usage: LINUX_MOUNT - to automatically mount BashBunny.
+#	 LINUX_UMOUNT - to unmount mounted BashBunny.
+
+function LINUX_MOUNT() {
+	Q DELAY 1000
```
Comment (rsxchin) [inline payloads/extensions/linux_mount.sh:10]: Why have a DELAY at the beginning of the function?

File: payloads/extensions/linux_mount.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	Q CTRL-ALT t
+	Q DELAY 1000
+	Q STRING unset HISTFILE
+	Q ENTER
+	Q DELAY 1000
```
Comment (rsxchin) [inline payloads/extensions/linux_mount.sh:15]: Why is this DELAY so long?

File: payloads/extensions/linux_mount.sh [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	Q STRING udisksctl mount -b /dev/'$disk'
+	Q ENTER
+	Q DELAY 2000
+	Q STRING lmnt='$(lsblk | grep $disk | awk '\'{print\ '$7'}\'\)''
+	Q ENTER
+	Q DELAY 500
+}
+function LINUX_UMOUNT() {
+	Q DELAY 1000
+	Q STRING udisksctl unmount -b /dev/'$disk'
```
Comment (rsxchin) [inline payloads/extensions/linux_mount.sh:28]: Why is there a DELAY at the beginning of th is function too?

File: payloads/extensions/linux_mount.sh (hunk 1 above)
Comment (drapl0n) [reply payloads/extensions/linux_mount.sh:10]: whenever Bunny is plugged in systems with desktop environment, it takes a second to process it as storage drive and prompt it in system tray.

File: payloads/extensions/linux_mount.sh (hunk 2 above)
Comment (drapl0n) [reply payloads/extensions/linux_mount.sh:15]: depending on various systems with different hardware configuration, it takes average time of a second to load terminal emulator.

Comment (rsxchin) [reply payloads/extensions/linux_mount.sh:15]: I suggest removing the starting DELAY here and allow the user the call their own DELAY in their payload before calling the function.

File: payloads/extensions/linux_mount.sh (hunk 3 above)
Comment (drapl0n) [reply payloads/extensions/linux_mount.sh:28]: payload between LINUX_MOUNT and LINUX_UMOUNT function may not be ended with a delay by author, in order to mitigate that delay has been added at beginning.

File: payloads/extensions/linux_mount.sh (hunk 2 above)
Comment (rsxchin) [reply payloads/extensions/linux_mount.sh:15]: This comment was asking why wait a full second after running "unset HISTFILE"

Comment (drapl0n) [reply payloads/extensions/linux_mount.sh:15]: yes I will remove that, I added that delay during testing phase.

File: payloads/extensions/linux_mount.sh (hunk 3 above)
Comment (rsxchin) [reply payloads/extensions/linux_mount.sh:28]: That should be handled in the payload

**Pull request merged**